examples/chef/chef.py

```diff
@@ -490,9 +492,17 @@ def main(argv: Sequence[str]) -> None:
     #
 
     if options.do_build:
-        branch = shell.run_cmd(
-            "git branch | awk -v FS=' ' '/\*/{print $NF}' | sed 's|[()]||g'", return_cmd_output=True).replace("\n", "")
-        commit_id = shell.run_cmd("git rev-parse HEAD", return_cmd_output=True).replace("\n", "")
+        if options.do_automated_test_stamp:
+            branch = shell.run_cmd(
+                "git branch | awk -v FS=' ' '/\*/{print $NF}' | sed 's|[()]||g'", return_cmd_output=True).replace("\n", "")
```

Comment:
Please do not shell out to "sed" and "awk" from a more readable general purpose programming language (Python).

Reply:
Hmm, it's great point, I'll parse the output inside python.

Reply:
Updated, thank you.